batch/batch/front_end/front_end.py

```diff
@@ -663,8 +693,52 @@ async def create_jobs(request: aiohttp.web.Request, userdata: dict):
         raise web.HTTPServiceUnavailable()
 
     batch_id = int(request.match_info['batch_id'])
+    user = userdata['username']
+    db: Database = app['db']
+
+    record = await db.select_and_fetchone(
+        '''
+SELECT token AS update_id FROM batches
```

Comment:
Not sure I understand why the token of the batch is the update_id.

Reply:
ah, I see. This is the token that we use for idempotency of the REST API, and subsequent updates will use their own distinct token.

Reply:
I think this is a backwards compatibility thing for old clients.

Reply:
The code path for new clients is the function below.

Reply:
How do we deprecate it? The old clients are going to use this endpoint.

Reply:
I just meant leave a comment so engineers know that this endpoint isn't going to be used in the future.

In the future we might want to do something where we add a "Deprecated" response header for deprecated endpoints which our python client can identify and then spit out a warning that the user might want to upgrade their hail installation. That's a separate issue though.